flask-api/API_endpoint.md

```diff
@@ -53,6 +53,40 @@ Will strip away both of the provided Cookies.
 
 Will require a valid JWT Cookie and the CSRF header (See login for CSRF example).
 
+## Voting Endpoints:
+
+### Create
+
+`/votes/create` methods=['POST']

```

Comment:
Read the requirements of this endpoint carefully. If it doesn't make sense, or needs updates, pls tell.